### PR TITLE
Minor adjustments.

### PR DESCRIPTION
**Pull request by ArthurSonzogni**

### CODE REVIEW

**Comment by ArthurSonzogni** [comment]
Hi,
I made some minor adjustment. Could you please take a look?

**Review by adi-g15 — APPROVED** [review]
@ArthurSonzogni  I find it all good sir, though I think instead of mentioning "ftxui" we can use the "${pkgname%-git}", as it is easier to just leave it as such even if package name is to be changed. Though since the name won't be changing, this is almost the same still.

Though, we do need to mention a version number in pkgver, even if "pkgver=1", as that is automatically updated when makepkg is run.
Also later on (quoting from Arch Wiki)-

> When releasing a new version of the packaged software, update the pkgver or pkgrel variables to notify all users that an upgrade is needed. Do not update those values if only minor changes to the PKGBUILD such as the correction of a typo are being published.

**Comment by adi-g15** [comment]
Also if you can approve it, then we should update .SRCINFO too.
The easiest i think is to run "makepkg && makepkg --printsrcinfo > .SRCINFO" when wanting to update package version in AUR, since it updates pkgver in PKGBUILD and .SRCINFO both.

File: PKGBUILD [hunk 1]
```diff
@@ -1,45 +1,42 @@
...
-depends=('cmake' 'gcc' 'ninja') # ninja is optional, just remove "-G Ninja" from build() too, though its faster if you leave as it is
-makedepends=('git')
-provides=("${pkgname%-git}")
-conflicts=("${pkgname%-git}")
-install='.install'
-source=("${pkgname%-git}::git+https://github.com/ArthurSonzogni/FTXUI")
-md5sums=('SKIP')
+license=("MIT")
+depends=()
+makedepends=("git" "gcc" "make")
```
Comment (adi-g15) [inline PKGBUILD]: @ArthurSonzogni you can remove "make" from makedepends.
\* I shouldn't have put the others too in depends though : |
[Quoting Arch wiki](https://wiki.archlinux.org/title/Makepkg#Usage): 

> Before continuing, install the base-devel group. **Packages belonging to this group are not required to be listed as build-time dependencies (makedepends)** in PKGBUILD files.

**Comment by adi-g15** [comment]
Also sir for pre-built packages, adding the repo following the step at https://github.com/adig-pkgs/adig-pkgs, pacman will now list the package - 

image

File: PKGBUILD (hunk 1 above)
Comment (ArthurSonzogni) [reply PKGBUILD]: Done. Thanks!

**Pull request merged**